webapp/packages/core-ui/src/Tabs/Tab/Tab.module.css

```diff
@@ -66,4 +66,7 @@
 
 .tabOuter {
   display: block;
+  border: 1px solid color-mix(in srgb, transparent, var(--theme-on-secondary) 10%);
+  border-top: none;
+  border-left: none;
```

Comment:
Hmm, this may affect vertical/horizontal tabs. Probably we should list it together where we are adding borders to tabs (it's not always there)